pkg/universe.dagger.io/go/build.cue

```diff
@@ -38,8 +41,22 @@ import (
 			}
 		}
 		command: {
+			_packages: [...string] | []
```

Comment:
Nit: `| []` should be useless

Reply:
clearly. Some debugging code I didn't remove…